app/indexers/data_indexer.rb

```diff
@@ -43,6 +47,12 @@ def legacy_collections
     end
   end
 
+  def virtual_object_constituents
+    return unless cocina.dro?
+
+    cocina.structural.hasMemberOrders.first&.members
```

Comment:
Can we be sure every DROStructural has a `hasMemberOrders`? Wondering if we need more safe navigation on this line. 🤷🏻‍♂️

Reply:
I'm pretty sure the tests cover this case. Every DROStructural will have hasMemberOrders and it will return an array: https://github.com/sul-dlss/cocina-models/blob/main/lib/cocina/models/dro_structural.rb#L7

Reply:
Sounds good.